conversion.py

```diff
@@ -2,3 +2,5 @@ def celsius_to_fahrenheit(celsius):
     return celsius/5*9 + 32
 
 
+def celsius_to_kelvin(celsius):
+    return celsius + 273.15
```

Comment:
```suggestion
    return celsius + 273.15

```
It's good to leave an empty line at the end of the file.